spp_change_request/models/change_request.py

```diff
@@ -45,11 +45,7 @@ class ChangeRequestBase(models.Model):
     _order = "id desc"
     _check_company_auto = True
 
-    def _default_name(self):
-        name = self.env["ir.sequence"].next_by_code("spp.change.request.num")
-        return name
-
-    name = fields.Char("Request #", required=True, default=_default_name)
```

Comment:
https://github.com/odoo/odoo/blob/f26dfd3724dc262c7f143873125976efb9b0ec8b/addons/stock/models/stock_production_lot.py#L20

why don't we use nameless function here? Instead of having default parameter set to "NEW" then change it in create function. This is what it does, as I imagine:
1. I create a new CR by pressing Create button in CR action, it open a form with name "NEW" as default.
2. I don't like the name "NEW", so I change it to something else, let say "Nhat's Change Request".
3. When press Save button, it override the name I carefully edited with next sequence from ir.sequence, don't even notice me anything.

I think, it is either we set the name is readonly, then we do anything we want as the account move, or we only do default